docs/build/build-guide.md

```diff
@@ -187,50 +196,16 @@ comparably minimal effort.
 
 #### Building Parachains with Cumulus
 
-Diving further into building parachains,
-[Cumulus](https://github.com/paritytech/polkadot-sdk/tree/master/cumulus) is another set of tools
-that aid in building a parachain-ready blockchain for Polkadot or Kusama. Cumulus utilizes FRAME and
-Substrate to create an easy way to build your first parachain. It ensures the chain follows the
-Parachain Protocol.
-
-:::info
-
-_Cumulus clouds are shaped like dots; together, they form an intricate, beautiful and functional
-system._
-
-:::
+[Cumulus](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/cumulus/index.html)
+transforms FRAME-based runtimes into Polkadot-compatible parachain runtimes, and Substrate-based
+nodes into Polkadot/Parachain-compatible nodes. In other words, Cumulus can enable a regular runtime
+to meet the requirements to have its state verified by Polkadot and take part in shared security.
 
 For most developers, the best place to start is to get familiar with Substrate independently,
 followed by FRAME, with Cumulus as the final step to understanding the entire parachain building
 process. This way, one can view how various paradigms are applied and decide on integrating or
 utilizing Substrate for their particular use case.
 
-Please see the [parachain development guide](build-parachains.md) for how to get started on building
-a parachain or parathread.
-
-#### Parachains Benefits
```

Comment:
Why are we getting rid of this section?

Reply:
It seemed repetitive; the parachain section in Learn should aim to "sell" people on the benefits of parachains. I was aiming to make this more purely focused on building as well as more concise.

Reply:
Sounds good. Is there a place on the Wiki where this content can be moved to?